tests/lax_metal_test.py

```diff
@@ -2605,15 +2605,16 @@ def testSearchsortedNans(self, dtype, side, method):
     xshape=[(20,), (5, 4)],
     binshape=[(1,), (5,)],
     right=[True, False],
+    method=['scan', 'scan_unrolled', 'sort', 'compare_all'],
     reverse=[True, False],
     dtype=default_dtypes,
   )
-  def testDigitize(self, xshape, binshape, right, reverse, dtype):
+  def testDigitize(self, xshape, binshape, right, method, reverse, dtype):
```

Comment:
I'd avoid modifying tests in this file. The metal backend is known to be very incomplete, and adding test cases here is a good way to inadvertently block your PR.